qa/tasks/cephfs/test_admin.py

```diff
@@ -2695,49 +2695,50 @@ def test_with_health_warn_trim(self):
 
     def _get_unhealthy_mds_id(self, health_warn):
         '''
-        Return MDS ID for which health warning in "health_warn" has been
-        generated.
+        Returns number of MDSs for which health warning in "health_warn" has been
+        generated and the first MDS ID for which warning is generated
         '''
         health_report = json.loads(self.get_ceph_cmd_stdout('health detail '
-                                                            '--format json'))
+                                                            '--format json-pretty'))
         # variable "msg" should hold string something like this -
         # 'mds.b(mds.0): Behind on trimming (865/10) max_segments: 10,
         # num_segments: 86
+        count = health_report['checks'][health_warn]['summary']['count']
         msg = health_report['checks'][health_warn]['detail'][0]['message']
         mds_id = msg.split('(')[0]
         mds_id = mds_id.replace('mds.', '')
-        return mds_id
+        return count, mds_id
 
-    def test_with_health_warn_with_2_active_MDSs(self):
+    def test_with_health_warn_on_1_mds_with_2_active_MDSs(self):
```

Comment:
Change of test name is unnecessary because both `test_with_health_warn_with_2_active_MDSs` are located in different classes. `TestFSFail.test_with_health_warn_with_2_active_MDSs` and `TestMDSFail.test_with_health_warn_with_2_active_MDSs`.

Reply:
ok. I missed that it's part of a separate class. The other reason to change the test name is to indicate what the test is really doing.